packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.tsx

```diff
@@ -81,8 +81,8 @@ export type ManageVersionsProps = {
 };
 
 export enum ManageVersionsTabs {
-  Versions = 0,
-  Changes = 1,
+  Versions = "NamedVersion",
```

Comment:
Isn't that a breaking change? If we revert this to a Minor, is this change needed ?

Reply:
Typescript enums like that comes with reverse mapping, which kinda makes their values "public api" ? (Also, potentially saved state of selected page will have saved `0`, not "NamedVersions".

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Yeah.. we expose..
Thats a good catch!!
Lets revert this to be with numbers, or there was an issue with it @DanishMehmood-bit ?

Reply:
technically.. its public if its exported in index, thats the case here...

Reply:
The Tabs component now accepts string as a value thats why I did this change, if I revert it back to numbers then I need to typecast the number into strings and vice versa for tab change.

Reply:
Please revert it and do the cast so we dont break as we want to change that to a Minor update.

Reply:
Reverted back to use numbers and added typecast